forem/forem
# Refactor:Remove jobs_url and Display Banner

## What type of PR is this? (check all applicable)


## Description
Remove the jobs_url and banner functionality when a user searches job or jobs since [we have decided that it will not be included in the Forem starter pack](https://docs.google.com/document/d/1facy0-UXlUPrzpll8TAM-MNjZiwVbO1vepphuE6a-ek/edit#). Also, I needed a good red PR today 💃🏻 

It was a good run code, https://github.com/forem/forem/pull/8259

## Related Tickets & Documents
closes: https://github.com/forem/forem/issues/13470

## QA Instructions, Screenshots, Recordings
1. Go to the homepage and search job or jobs and observe that the banner is no longer present
2. Go to `admin/config` and observe that the "Jobs" section has been removed. 

## Added tests?
- [x] No bc I am removing code

## How will this change be communicated?
      or in a [forem.dev](http://forem.dev) post


![good run](https://media4.giphy.com/media/oWsgQULaD4gdHU1CwX/200.gif?cid=95b279441159a7d1186b1374ee38d62c192a45bec074cbc9&rid=200.gif&ct=g)
